.env.testnet.template

```diff
@@ -0,0 +1,24 @@
+FAUCET_ADDRESS=https://testnet-faucet.onflow.org/fund-account
+
+# These values will be set by the startup script.
+# See startup.js
+ADMIN_ADDRESS=""
+FLOW_PRIVATE_KEY=""
+FLOW_PUBLIC_KEY=""
+
+DATABASE_PATH=./kitty-items-db-testnet.sqlite
+
+EXT_FLOW_TOKEN_ADDRESS=0x7e60df042a9c0868
+EXT_FUNGIBLE_TOKEN_ADDRESS=0x9a0766d93b6608b7
+EXT_NON_FUNGIBLE_TOKEN_ADDRESS=0x631e88ae7f1d7c20
+EXT_NFT_STOREFRONT_ADDRESS=0x94b06cfca1d8a476
+EXT_METADATA_VIEWS_ADDRESS=0x631e88ae7f1d7c20
```

Comment:
What does the `EXT` prefix mean here?

Reply:
We have to call these something different else we'll have an infinite loop when expanding the values in the project's `.env.local. `EXT` is just arbitrary prefix.